data/json/recipes/nested.json

```diff
@@ -545,7 +545,12 @@
     "name": "chitinous leg guards",
     "description": "Recipes related to constructing durable leg guards from chitin.",
     "skill_used": "tailor",
-    "nested_category_data": [ "legguard_chitin", "legguard_acidchitin" ],
+    "nested_category_data": [ 
+      "legguard_chitin", 
+      "legguard_acidchitin",
+      "xs_legguard_acidchitin",
+      "xl_legguard_acidchitin"
+    ],
```

Comment:
```suggestion
    "nested_category_data": [ "legguard_chitin", "legguard_acidchitin", "xs_legguard_acidchitin", "xl_legguard_acidchitin" ],
```

Reply:
Nope, this is just a project-wide formatting rule for line length. If the JSON array is short enough, the linter will insist it stays in one line. For the other arrays, they are too long to be formatted on one line. The JSON style check is picky about it, but it keeps the formatting fairly consistent

Reply:
Could I go around this by simply adding the chitin XS and XL versions to the nested group in this PR (along with the items and recipes associated), so my PR becomes compliant due to having the same length as the other groups?

Reply:
Yes, if you add more things it will likely form the array as it was before.  Some nested groups with short line length just take up that single line.  It isn't a big deal either way so just use the auto-linter, (http://dev.narc.ro/cataclysm/format.html) and paste the results.